scripts/build-server

```diff
@@ -8,6 +8,6 @@ cd $(dirname $0)/..
 mkdir -p bin
 [ "$(uname)" != "Darwin" ] && LINKFLAGS="-extldflags -static -s"
 
-RKE_VERSION=$(grep '^github.com/rancher/rke' vendor.conf | awk '{print $2}')
+RKE_VERSION=$(grep -m1 '^github.com/rancher/rke' vendor.conf | awk '{print $2}')
```

Comment:
Why was this added? We should only have 1 version in vendor.conf

Reply:
For development & testing purposes it is helpful to have more than one, where a dev version may be defined at the top of vendor.conf. As trash selects the first version, placing the dev version at the top will avoid conflicts if rebasing.